yarn-project/end-to-end/package.json

```diff
@@ -77,12 +82,35 @@
     "winston": "^3.10.0"
   },
   "devDependencies": {
-    "concurrently": "^7.6.0"
+    "@jest/globals": "^29.5.0",
+    "@types/jest": "^29.5.0",
+    "concurrently": "^7.6.0",
+    "jest": "^29.5.0",
+    "ts-node": "^10.9.1",
+    "typescript": "^5.0.4"
   },
   "files": [
     "dest",
     "src",
     "!*.test.*"
   ],
-  "types": "./dest/index.d.ts"
+  "types": "./dest/index.d.ts",
+  "engines": {
+    "node": ">=18"
+  },
+  "jest": {
+    "extensionsToTreatAsEsm": [
+      ".ts"
```

Comment:
We got an .mjs in the end-to-end tests, do you know if we need to flag it here?

Reply:
If CI passes, I'm going to assume not relevant. I mean this was a stack-overflow paste so I don't really have a deep understanding of why I even had to add it. But it made an error go away.
Hate this stuff. That's why I was investigating `bun`. Maybe one day.